foundry.toml

```diff
@@ -7,7 +7,7 @@ solc_version = "0.8.28"
 evm_version = "cancun"
 
 optimizer = true
-optimizer_runs = 90
+optimizer_runs = 1
```

Comment:
Why this? Hub size issues?

Reply:
Not sure where this comes from. I didnt change anything in the toml.

Reply:
Ohh, it comes from a @hieronx PR. Is this a local testing artifact, Jeroen?

Reply:
It was while trying to get the `Hub` contract below the limit again. Not sure it's still needed. Let's just add a ticket to update this to the highest possible value just before audit?

Reply:
https://github.com/Recon-Fuzz/centrifuge-review-protocol-v3/issues/20